### PR TITLE
[Testing] Enabling ported UITests from Xamarin.UITests to Appium - 7

### PR DESCRIPTION
**Pull request by ghost**
### Description of Change

This PR focuses on re-enabling and updating following tests from the Xamarin.UITest framework to be compatible with Appium. The tests, previously commented out, are reviewed, and modified to ensure they are functional with the Appium framework. We are going to enable tests in blocks in different PRs. This is the seventh group of tests enabled.

### TestCases

- **Issue2993**
- **Issue3008**
- **Issue3019**
- **Issue3053**
- **Issue3012**

### CODE REVIEW

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue2993.cs [hunk 1]
```diff
@@ -13,11 +12,10 @@ public Issue2993(TestDevice testDevice) : base(testDevice)
...
-	// 	App.WaitForElement("Bottom Text");
-	// }
-}
-#endif
\ No newline at end of file
+	[Test]
+	[Category(UITestCategories.Layout)]
+	public void BottomContentVisibleWithBottomBarAndNavigationPage()
+	{
+		App.WaitForElement("BottomText");
```
[the remark this answers is not in the document]
Comment (anandhan-rajagopal) [reply src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue2993.cs:19]: Yes, WaitForElement can locate elements by text on all platforms except Windows. On Windows, it results in a timeout exception, and in this case, setting the AutomationID is works.

File: src/Controls/tests/TestCases.HostApp/Issues/XFIssue/Issue2993.cs [hunk 2]
```diff
@@ -35,7 +35,7 @@ protected override void Init()
 		Children.Add((createPage()));
 		Children.Add(new ContentPage()
 		{
-			IconImageSource = "calculator.png",
+			IconImageSource = "coffee.png",
```
[the remark this answers is not in the document]
Comment (anandhan-rajagopal) [reply src/Controls/tests/TestCases.HostApp/Issues/XFIssue/Issue2993.cs]: Yes, I got the images from Xamarin.Forms gallery and committed the changes

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue2993.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (jsuarezruiz) [reply src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue2993.cs]: Ok, applied changes to fix it!

**Review by jsuarezruiz — DISMISSED** [review]

**Review by PureWeen — DISMISSED** [review]
Fix merge conflicts

**Review by jsuarezruiz — DISMISSED** [review]

**Review by PureWeen — DISMISSED** [review]

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue3012.cs [hunk 3]
```diff
@@ -1,31 +1,29 @@
...
-using NUnit.Framework;
+﻿using NUnit.Framework;
 using UITest.Appium;
 using UITest.Core;
 
 namespace Microsoft.Maui.TestCases.Tests.Issues;
 
 public class Issue3012 : _IssuesUITest
 {
+	const string DumbyEntry = "DumbyEntry";
```
[the remark this answers is not in the document]
Comment (anandhan-rajagopal) [reply src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue3012.cs]: Yeah, replaced the `DumbyEntry` with `OtherEntry` and changes committed. Kindly review and share your concerns.

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue3019.cs [hunk 4]
```diff
@@ -12,16 +12,13 @@ public Issue3019(TestDevice testDevice) : base(testDevice)
...
-	// 	App.WaitForElement("Group 1");
-
-	// 	App.Tap(x => x.Marked("Grouped Item: 0"));
-	// 	App.Tap(x => x.Marked("Grouped Item: 1"));
-	// 	App.Tap(x => x.Marked("Grouped Item: 1 Clicked"));
-
-	// }
+	[Test]
+	[Category(UITestCategories.ListView)]
+	public void MakeSureListGroupShowsUpAndItemsAreClickable()
```
Comment (jsuarezruiz) [inline src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue3019.cs:17]: This test is failing on Android:
image

**Review by jsuarezruiz — DISMISSED** [review]

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue3019.cs (hunk 4 above)
Comment (anandhan-rajagopal) [reply src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue3019.cs:17]: Fixed the test failure and committed the changes.
image

**Review by jfversluis — APPROVED** [review]

**Pull request merged**